test/integration/cross.css.and.style.api.test.ts

```diff
@@ -93,7 +93,7 @@ describe('Verify interaction between the CSS and style APIs', () => {
         kind: ShapeBpmnElementKind.EVENT_END,
         label: 'message end 2',
         stroke: { color: strokeColor },
-        verticalAlign: <VerticalAlign>'top', // the element has a label
+        verticalAlign: 'top' as VerticalAlign, // the element has a label
```

Comment:
**suggestion:** I guess that the explicit type can be removed here, as the object is explicitly a `ExpectedShapeModelElement` whose the type of the `verticalAlign` propety is `VerticalAlign`. I tested this locally.

This remark applies to the whole file.